quirks/password-rules.json

```diff
@@ -102,6 +102,10 @@
     "girlscouts.org": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$#!];"
     },
+    "gov.br": {
+        "password-rules": "minlength: 8; maxlength: 70; required: [!"#$%&()*+,-./:;<=>?@[\]^{|}~.];"
```

Comment:
I don’t think you can include a quote here like this. This needs to be escaped, somehow.

Reply:
edited to match actua rules.